docs/explanation/cluster-maintenance.rst

```diff
@@ -28,7 +28,7 @@ Enabling maintenance mode
 
 - Check if OSDs on the node are ``ok-to-stop`` to ensure sufficient redundancy to tolerate the loss
   of OSDs on the node.
-- Check if the number of running services is greater than the minimum (3 MON, 1 MDS, 1 MGR)
+- Check if the number of running services is greater than the minimum (majority of MON, 1 MDS, 1 MGR)
```

Comment:
Is this criteria modification done so that a minimal functional ceph cluster is available when a particular node is going into maintenance ?

Reply:
No, this modification only focuses on the mon services. But I agree that a proper check can be implemented for MDS / MGR too, would bug reports sufficient for now?

Reply:
yeah, a bug will help keep track of status. Something like `ensure FS (mds) availability during maintenance`.